libs/platform/src/lib/components/search-field/search-field.component.html

```diff
@@ -1,106 +1,62 @@
-<div
-    class="search-field"
-    [ngClass]="{
+<div class="search-field" [ngClass]="{
         'with-categories': showCategoryDropdown,
         'is-loading': isLoading,
         'hide-category-label': hideCategoryLabel
-    }"
->
-    <div
-        class="search-field--input-group fd-input-group"
-        [attr.aria-controls]="menuId"
-        [attr.aria-expanded]="showDropdown ? 'true' : 'false'"
-        aria-haspopup="true"
-        #inputGroup
-    >
-        <span
-            class="fd-input-group__addon fd-input-group__addon--button"
-            [ngClass]="{ 'fd-input-group__addon--compact': compact }"
-            *ngIf="showCategoryDropdown"
-        >
-            <button
-                class="search-field--category-button fd-input-group__button fd-button fd-button--menu"
-                [ngClass]="{ 'fd-button--compact': compact }"
-                [fdpMenuTriggerFor]="categoryMenu"
-            >
-                <span class="search-field--category-label" *ngIf="!hideCategoryLabel">{{
+    }">
+    <div class="fdp-search-field__input-group fd-input-group" [attr.aria-controls]="menuId"
+        [attr.aria-expanded]="showDropdown ? 'true' : 'false'" aria-haspopup="true" #inputGroup>
```

Comment:
Can we put ` #inputGroup` at the beginning of the tag like other lines ?

Reply:
This has been fixed.